inc/functions/files.php

```diff
@@ -1273,24 +1273,23 @@ function rocket_get_filesystem_perms( $type ) {
 }
 
 /**
- * Try to find the correct wp-config.php file, support one level up in filetree
+ * Try to find the correct wp-config.php file, support one level up in file tree.
  *
  * @since 2.1
  *
- * @return string|bool The path of wp-config.php file or false if not found
+ * @return string|bool The path of wp-config.php file or false if not found.
  */
 function rocket_find_wpconfig_path() {
 	/**
-	 * Filter the wp-config's filename
+	 * Filter the wp-config's filename.
 	 *
 	 * @since 2.11
-	 * @author Maxime Culea
 	 *
 	 * @param string $filename The WP Config filename, without the extension.
 	 */
 	$config_file_name = apply_filters( 'rocket_wp_config_name', 'wp-config' );
-	$config_file      = ABSPATH . $config_file_name . '.php';
-	$config_file_alt  = dirname( ABSPATH ) . '/' . $config_file_name . '.php';
+	$config_file      = rocket_get_constant( 'ABSPATH' ) . $config_file_name . '.php';
```

Comment:
1. Store the constant in a var and reuse to avoid calling it 2x.
2. Store the Filesystem handler in a var to avoid calling `rocket_direct_filesystem()` more than 1x.